openeo-geotrellis/src/test/scala/org/openeo/geotrellis/layers/FileLayerProviderTest.scala

```diff
@@ -1298,6 +1298,49 @@ class FileLayerProviderTest extends RasterMatchers{
     }
   }
 
+  @Test
+  def testAntimerideanArtifacts(): Unit = {
+    val projectedExtent = ProjectedExtent(Extent(300000, 7690200, 409800, 7800000), CRS.fromName("EPSG:32601"))
+    val poly2 = ProjectedPolygons(Array(MultiPolygon(projectedExtent.extent.toPolygon())), projectedExtent.crs)
+
+    val txt = Source.fromResource("org/openeo/geotrellis/testAntimerideanArtifacts.json").mkString
+    val mockedFeatures = CreoFeatureCollection.parse(txt)
+    val client = new MockOpenSearchFeatures(mockedFeatures.features)
+
+    val inputFile = "/eodata/Global-Mosaics/Sentinel-1/S1SAR_L3_IW_MCM/2020/08/01/Sentinel-1_IW_mosaic_2020_M08_60WWB_1_0/VV.tif"
+    val basePathArtifactory = "https://artifactory.vgt.vito.be/artifactory/testdata-public"
+    // copy to local /eodata. YOu might need to create /eodata first:
```

Comment:
@EmileSonneveld  do you have a pointer to where we depend on /eodata?

Reply:
Here some code requires the files to be under `/eodata`:
https://github.com/Open-EO/openeo-opensearch-client/blob/bf6cab22d67def734303dcee623ef39a1419ce6d/src/main/scala/org/openeo/opensearch/OpenSearchResponses.scala#L881

On Jenkins, the folder structure seems to be created automatically with copyURLToFile, but on my computer I had to explicitly create it

Reply:
Test is changed to not write to /eodata anymore